lib/api.js

```diff
@@ -93,6 +93,17 @@ var api = {
   getAccessToken: function() {
     return require('./auth').getAccessToken(refreshToken, commandScopes);
   },
+  addAccessTokenToHeader: function(reqOptions) {
+  // Runtime fetch of Auth singleton to prevent circular module dependencies
+    var auth = require('../lib/auth');
+
+    return auth.getAccessToken(refreshToken, commandScopes).then(function(result) {
+      reqOptions.headers = {
```

Comment:
maybe this should be `_.set(reqOptions, 'headers.authorization', 'Bearer ' + result.access_token)` so if there're any headers on the reqOptions they don't get overwritten